src/main/java/com/openlattice/organization/OrganizationsApi.java

```diff
@@ -258,4 +260,13 @@ Void removeRoleFromUser(
             @Path( ROLE_ID ) UUID roleId,
             @Path( USER_ID ) String userId );
 
+    //Admin-only APIs
+    @POST( BASE + ID_PATH + CONNECTIONS )
+    Void addConnection( @Path( ID ) UUID organizationId, @Body Set<String> connections );
```

Comment:
Will this endpoint add multiple connections if `Set<String> connections` has multiple values? If yes, can this be renamed to `addConnections`?